driftbase/api/events.py

```diff
@@ -73,14 +81,25 @@ def post(self):
             else:
                 event["player_id"] = player_id  # Always override!
             eventlogger.info("eventlog", extra={"extra": event})
-
+        
         if get_feature_switch('enable_eventlog_shoutout_forwarding') and is_service:
             events_to_shoutout = defaultdict(list)
-            for event in [e for e in events if not e.get('event_name', '').startswith('drift.')]:
+            events_blacklist = get_tenant_config_value('eventlog', 'events_blacklist',
+                                                       defaults=default_eventlog_config)
+            
+            
+            for event in events:
+                if (
+                    (event_name := event.get("event_name"))
+                    and events_blacklist
+                    and any(fnmatch.fnmatch(event_name, blocked_event) for blocked_event in events_blacklist)
```

Comment:
might want to rename `events_blacklist` to `events_blocklist`, to match the variable naming.